translations/messages-en.properties

```diff
@@ -979,5 +979,5 @@ instance.upgrade.confirm.warning.text = This action cannot be undone, make sure
 confirm.delete.progress = Deletion in progress
 confirm.delete.totals = {{totalDeleted}}/{{totalSelected}} deleted...
 messages.unknown.sender = Unknown sender
-messages.sent.by = Sent by {{senderName}}
+messages.sent.by = Submitted by {{senderName}}
```

Comment:
Unfortunately this won't work - the next time the messages are exported from our collaboration tool the default values will be overwritten. Follow [this documentation](https://github.com/medic/medic-docs/blob/master/development/translations.md#modifying-any-existing-translation-values) to update the messages directly in POE. If you don't have access to POE get a login from @abbyad .

Reply:
I've reverted the translation change and edited the tests so they are translation-independent.

Reply:
Would you like me to update POE and export so we can get this one merged?